commons/src/main/java/org/infinispan/commons/dataconversion/EncodingUtils.java

```diff
@@ -4,6 +4,7 @@
  * Utilities to encode/decode keys and values from caches.
  *
  * @since 9.1
+ * @deprecated Use the org.infinispan.encoding.DataConversion obtained from the AdvancedCache.
  */
 public final class EncodingUtils {
```

Comment:
I deprecated because this class is public, it's not used because I already removed the (internal) usages